client/network/src/service/signature.rs

```diff
@@ -0,0 +1,30 @@
+use super::*;
```

Comment:
Missing license

Reply:
Also, could you add:

```suggestion
pub use libp2p::identity::PublicKey;
```

So that users of `sc_network` don't have to depend themselves on (the correct version of) `libp2p`

Reply:
> Missing license

We have it in Cargo.toml and on the workspace, do you know why do we need it in every file?

> pub use libp2p::identity::PublicKey;

There are so many different things called `PublicKey` that we will confuse them for sure. You meant that dependents of the `sc-network` crate could refer to the libp2p one by `sc_network::PublicKey`, right?

Reply:
Yes